eng/utils.ps1

```diff
@@ -260,20 +287,22 @@ function install-llvm {
         [string]$qirlibDir,
         [Parameter(Mandatory)]
         [ValidateSet("download", "build")]
-        [string]$operation
+        [string]$operation,
+        [Parameter(Mandatory)]
+        [ValidateSet("llvm11-0", "llvm12-0", "llvm13-0")]
```

Comment:
The readme mentions that qirlib can compile with LLVM 14, so is it worth adding that to the set or should a version 14 user expect to point to a manually installed/compiled version?